topics/benchmark.md

```diff
@@ -63,6 +63,16 @@ a default set of tests, or you can supply a custom set of tests.
   mode, the key must contain "{tag}". Otherwise, the
   command will not be sent to the right cluster node.
 
+**`--rfr`** _mode_
+: Enable read from replicas in cluster mode.
+  This command must be used with the `--cluster` option.
+  There are three modes for reading from replicas:
+  _no_ - sends read requests to primaries only (default).
+  _yes_ - sends read requests to replicas only.
+  _all_ - sends read requests to all nodes.
+   Since write commands will not be accepted by replicas,
+   it is recommended to enable read from replicas only for read command tests.
+
```

Comment:
```suggestion
**`--rfr`** _mode_
: Enable read from replicas in cluster mode.
  This command must be used with the `--cluster` option.
  There are three modes for reading from replicas:
  
  **no** - sends read requests to primaries only (default).
  
  **yes** - sends read requests to replicas only.
  
  **all** - sends read requests to all nodes.
  
   Since write commands will be rejected by replicas,
   it is recommended to enable read from replicas only for read command tests.

```

nit